website/docs/registry/providers/publishing.mdx

```diff
@@ -54,7 +54,7 @@ It supports the following fields:
 | Field                        | Purpose                                                                                                                                                                                                                                                                                                                        |
 | ---------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
 | `metadata`                   | Object containing additional data to be associated with a release.                                                                                                                                                                                                                                                             |
-| `metadata.protocol_versions` | List of strings with supported Terraform protocol versions of the provider. Providers developed using [Terraform Plugin SDK version 2](/plugin/sdkv2) should set this to `["5.0"]`. Providers developed using [Terraform Plugin Framework](/plugin/framework) should set this to `["6.0"]`. Defaults to `["5.0"]`. |
+| `metadata.protocol_versions` | List of strings with supported Terraform protocol versions of the provider. Providers developed using [Terraform Plugin SDK version 2](/plugin/sdkv2) should set this to `["5.0"]`. Providers developed using [Terraform Plugin Framework](/plugin/framework) should set this to `["6.0"]`, unless the Framework is [explicitly configured for protocol version 5](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-framework/providerserver#ServeOpts.ProtocolVersion). Defaults to `["5.0"]`. |
```

Comment:
```suggestion
| `metadata.protocol_versions` | List of strings with supported Terraform protocol versions of the provider. Providers developed using [Terraform Plugin SDK version 2](/plugin/sdkv2) should set this field to `["5.0"]`. Providers developed using [Terraform Plugin Framework](/plugin/framework) should set this field to `["6.0"]` unless the Framework is [explicitly configured for protocol version 5](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-framework/providerserver#ServeOpts.ProtocolVersion). Defaults to `["5.0"]`. |
```

Suggest adding "field" to both instances of "set this" to avoid using "this" without a subject.